utils/handle_status_command.go

```diff
@@ -0,0 +1,73 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+)
+
+type EthCallResponse struct {
+	Jsonrpc string `json:"jsonrpc"`
+	Id      int    `json:"id"`
+	Result  string `json:"result"`
+}
+
+// updatedMsg := fmt.Sprintf("Token 0: %s Token 1: %s", unclaimed_fees_token_0, unclaimed_fees_token_1)
+func HandleStatusCmd() string {
+	final_result := ""
+	for k, v := range RegisteredPools {
+		var jsonStr = fmt.Sprintf(`
+			{
+				"jsonrpc": "2.0",
+				"id": 1,
+				"method": "eth_call",
+				"params": [
+					{
+						"from": "%s",
+						"to": "%s",
+						"data": "%s"
+					},
+					"latest"
+				]
+			}
+		`, v.from, v.to, v.data)
+		requestBody := bytes.NewBuffer([]byte(jsonStr))
+		API_URL := "https://mainnet.infura.io/v3/099fc58e0de9451d80b18d7c74caa7c1"
```

Comment:
Geth node URL should be set in main.go (flags and environment), not hardcoded here